src/harness/testcases/WINNF_FT_S_IPR_testcase.py

```diff
@@ -0,0 +1,220 @@
+from test_harness_objects import DomainProxy
+from full_activity_dump_helper import getFullActivityDumpSasTestHarness, getFullActivityDumpSasUut
+from sas_test_harness import SasTestHarnessServer, generateCbsdRecords
+
+LOW_FREQUENCY_LIMIT_HZ = 3550000000
+HIGH_FREQUENCY_LIMIT_HZ = 3650000000
+
+
+class FederalIncumbentProtectionTestcase(sas_testcase.SasTestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  def frequencyInBand(self, low_frequency, high_frequency):
+    """Returns True iff the given range overlaps with the 3550MHz to 3650MHz range."""
+    return (LOW_FREQUENCY_LIMIT_HZ <= low_frequency <= HIGH_FREQUENCY_LIMIT_HZ
+            or
+            LOW_FREQUENCY_LIMIT_HZ <= high_frequency <= HIGH_FREQUENCY_LIMIT_HZ)
+
+  def generate_IPR_1_default_config(self, filename):
+    """Generates the WinnForum configuration for IPR_1"""
+
+    # Load Devices
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+
+    # Pre-load conditionals and remove reg conditional fields from registration
+    # request.
+    conditional_keys = [
+        'cbsdCategory', 'fccId', 'cbsdSerialNumber', 'airInterface',
+        'installationParam', 'measCapability'
+    ]
+    reg_conditional_keys = [
+        'cbsdCategory', 'airInterface', 'installationParam', 'measCapability'
+    ]
+    conditionals_b = {key: device_b[key] for key in conditional_keys}
+    conditionals_c = {key: device_c[key] for key in conditional_keys}
+    device_b = {
+        key: device_b[key]
+        for key in device_b
+        if key not in reg_conditional_keys
+    }
+    device_c = {
+        key: device_c[key]
+        for key in device_c
+        if key not in reg_conditional_keys
+    }
+
+    # Load grant requests.
+    grant_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+
+    sas_test_harness_config = {
+        'sasTestHarnessName':
+            'SAS-Test-Harness-1',
+        'hostName':
+            'localhost',
+        'port':
+            9001,
+        'serverCert':
+            os.path.join('certs', 'server.cert'),
+        'serverKey':
+            os.path.join('certs', 'server.key'),
+        'caCert':
+            os.path.join('certs', 'ca.cert'),
+        'fullActivityDumpRecords': [
+            generateCbsdRecords([device_a], [[grant_a]])
+        ]
+    }
+
+    dpa_1 = {
+       'dpaId': 'east_dpa_4',
+       'frequencyRange': {'lowFrequency': 3550000000, 'highFrequency': 3650000000}
```

Comment:
Add a new keyword:
```
'points_builder': 'default (25, 10, 10, 10)'
```
This will use the default method for creating about 50 points in the DPA. In practice, this will be used to specify a 
file name holding a GeoJson file with a MultiPoint geometry.

Reply:
Done.